lib/elixir/lib/dynamic_supervisor.ex

```diff
@@ -564,7 +565,8 @@ defmodule DynamicSupervisor do
 
   """
   @doc since: "1.6.0"
-  @spec init([init_option]) :: {:ok, sup_flags()}
+  @spec init([init_option]) ::
+          {:ok, sup_flags() | (old_erlang_sup_flags :: :supervisor.sup_flags())}
```

Comment:
It does not return old_erlang_sup_flags. :)

Reply:
fixed